index.html

```diff
@@ -839,33 +841,33 @@
               <var>acceptPromise</var> with that error and terminate this
               algorithm.
               </li>
-              <li>Determine which <a>payment handlers</a> support
-              <var>identifier</var>. For each resulting payment handler, if
-              payment method specific capabilities supplied by the payment
-              handler match those provided by <var>data</var>, the payment
-              handler matches.
+              <li>Add each <a>payment handler</a> that supports
```

Comment:
It would be disingenuous the mention `.data` here, because the steps before this one already rejected bad data explicitly with an error. Let's please avoid making statements about how things work without solid proof that they work that way. 

Hypothetical "well, another user agent might maybe oneday kinda sorta do it differently" strawpersons are extremely unhelpful. Let's please stick to documenting reality.